config/oidc-mock-clients.json

```
@@ -14,6 +14,9 @@
   "RedirectUris": [
       "http://localhost:3000"
   ],
+  "PostLogoutRedirectUris" : [
+      "http://localhost:300"
```

Q: Ist das korrekt so?
```suggestion
      "http://localhost:3000"
```